pecl-manager/providers/script.rb

```diff
@@ -29,14 +29,21 @@
       :envvar_file => new_resource.envvar_file,
       :envvar_json => envvar_json,
       :gearman_user => node['pecl-manager']['user']
-    )
+              )
   end
 
-  service 'pecl-manager' do
-    supports [:start, :stop, :restart, :status]
-    action :restart
+  if node.attribute?('vagrant')
+    service 'pecl-manager' do
+      supports [:start, :stop, :restart, :status]
+      action :restart
+      only_if 'test -f /vagrant_gearman/vendor/autoload.php'
```

Comment:
We do have multiple vagrantboxes using this (import-export for example). Maybe add `ignore_failure true` if it is in vagrant, instead of using `only_if`?